ui/src/containers/AppDetailsContainer.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import AppsTable from '../components/AppsTable';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { getApps, reverseAppDetailsTableSort } from '../actions/actions-ui';
+
+export class AppDetailsContainer extends React.Component {
+  constructor (props) {
+    super(props);
+    this.onSort = this.onSort.bind(this);
+    this.onRowClick = this.onRowClick.bind(this);
+    this.getTable = this.getTable.bind(this);
+  }
+
+  componentWillMount () {}
+
+  onRowClick (event, rowId, props) {
+    var app = this.props.apps.data.filter((app) => {
+      return app.appName === rowId[0];
+    });
+    const id = app[0].id;
+    const path = '/app/' + id;
+    this.props.history.push(path);
```

Comment:
this came in from a previous pr, but as we're not utilising the history for navigation I think we can probably remove it from here (and also the other container)

Reply:
Hi @damienomurchu we are using this here. And we can also use it to do navigation on the header. I've tried this out and it stores the history of the navigation.

Reply:
It should be removed from the AppDetailsContainer but not from the AppsTableContainer.